pre_commit/commands/hook_impl.py

```diff
@@ -231,6 +232,8 @@ def _run_ns(
         return _ns(hook_type, color, is_squash_merge=args[0])
     elif hook_type == 'post-rewrite':
         return _ns(hook_type, color, rewrite_command=args[0])
+    elif hook_type == 'pre-rebase':
+        return _ns(hook_type, color, from_ref=args[0], to_ref=args[1])
```

Comment:
the docs seem to indicate that _sometimes_ this has 2 args, sometimes it has 1 -- I'm not sure that `from-ref` and `to-ref` arguments are correct here (they're meant for `pre-push`)?

Reply:
Args changed, let me know if naming are OK